flow/cmd/logged_interceptor.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"context"
+
+	"go.temporal.io/sdk/interceptor"
+	"go.temporal.io/sdk/workflow"
+)
+
+type LoggedWorkflowInboundInterceptor struct {
+	interceptor.WorkflowInboundInterceptorBase
+	Next interceptor.WorkflowInboundInterceptor
+}
+
+func NewLoggedWorkflowInboundInterceptor(next interceptor.WorkflowInboundInterceptor) *LoggedWorkflowInboundInterceptor {
+	return &LoggedWorkflowInboundInterceptor{
+		WorkflowInboundInterceptorBase: interceptor.WorkflowInboundInterceptorBase{Next: next},
+		Next:                           next,
+	}
+}
+
+func (w *LoggedWorkflowInboundInterceptor) ExecuteWorkflow(
+	ctx workflow.Context,
+	in *interceptor.ExecuteWorkflowInput,
+) (interface{}, error) {
+	// Workflow starts here
+	result, err := w.Next.ExecuteWorkflow(ctx, in)
+	// Workflow ends here
+	return result, err
```

Comment:
```suggestion
	return w.Next.ExecuteWorkflow(ctx, in)
```

Reply:
Just experimental, we are not actually using it right now